config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "kaholo-plugin-json",
+  "name": "json",
   "viewName": "Kaholo JSON Plugin",
```

Comment:
Shouldn't the `viewName` be changed as well?

Reply:
fixed